nifi-assembly/LICENSE

```diff
@@ -2834,3 +2834,25 @@ The binary distribution of this product bundles 'JUnit Platform' and 'JUnit Jupi
   notice.
 
   You may add additional accurate notices of copyright ownership.
+
```

Comment:
Looks like MVEL also bundles (?) ASM as per https://github.com/mvel/mvel/blob/master/ASM-LICENSE.txt.  Not sure if that is being consumed imported, but wanted to flag it.

Reply:
The MVEL library isn't used directly in this tooling, however it is included in easy rules library, via the easy-rules-mvel dependency and it does use Easy Rules MVELCondition objects.  Will this dependency still need to be cited?

Reply:
I don't see it in the dependency tree:

```|  +- org.jeasy:easy-rules-core:jar:3.3.0:compile
[INFO] |  +- org.jeasy:easy-rules-mvel:jar:3.3.0:compile
[INFO] |  |  +- org.jeasy:easy-rules-support:jar:3.3.0:compile
[INFO] |  |  \- org.mvel:mvel2:jar:2.4.3.Final:compile
[INFO] |  \- org.jeasy:easy-rules-spel:jar:3.3.0:compile
[INFO] |     +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.8:compile
[INFO] |     |  \- com.fasterxml.jackson.core:jackson-core:jar:2.9.8:compile
[INFO] |     +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.8:compile
[INFO] |     |  \- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.0:compile
[INFO] |     \- org.springframework:spring-expression:jar:4.3.23.RELEASE:compile
[INFO] |        \- org.springframework:spring-core:jar:4.3.23.RELEASE:compile
[INFO] |           \- commons-logging:commons-logging:jar:1.2:compile
[INFO]```

Reply:
@mattyb149 chatted with @apiri offline. It looks like MVEL is a transitive dependency so I just added license info for it